apps/chat/src/utils/app/publications.ts

```diff
@@ -504,3 +529,23 @@ export const mapFilterToRule = (
   source: filter.id,
   targets: filter.filterParams,
 });
+
+export const regenerateApiKeyNameAndVersionParts = (
+  entityId: string,
+  name: string,
+  version: string,
+): string => {
+  const preparedName = prepareEntityName(name);
+
+  if (isConversationId(entityId)) {
+    const modelName = splitEntityId(entityId).name;
+    const parsedModelReference = parseConversationApiKey(modelName).model.id;
```

Comment:
I would suggest to use `parsedModelId` name here
```suggestion
    const parsedModelId = parseConversationApiKey(modelName).model.id;
```

Reply:
are you sure? Actually we store reference inside `model.id`